atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -140,24 +137,16 @@ public void checkAgainstAccidentalTableAPIChanges() throws IOException {
         Schema schema = ApiTestSchema.getSchema();
         schema.renderTables(testFolder.getRoot());
 
-        List<String> generatedTestTables = ApiTestSchema.getSchema().getAllTables().stream()
-                .map(entry -> entry.getTableName() + "Table")
-                .collect(Collectors.toList());
+        String schemaName = ApiTestSchema.getSchema().getName();
+        checkIfFilesAreTheSame(schemaName + "TableFactory");
```

Comment:
Previously this test was not testing the generated table factory. This explains why tests didn't break in https://github.com/palantir/atlasdb/pull/5843.

I've updated this test to ensure we're testing all generated files.